apps/azure-functions-api/README.md

```diff
@@ -0,0 +1,78 @@
+# Azure Function Template
+
+This Workspace is a template to implement an Azure Function for simple REST API.
+
+## Architecture
+
+The project is structured as follows:
+
+```
+azure-functions-api
+|-- api
+|-- src
+|   |-- functions
+|   |-- generated
+|   |-- middlewares
+|   |-- models
+|   |-- utils
+|   |   |-- cosmos
+|   |   |-- redis
+|   |-- config.ts
+|   |-- main.ts
+|-- package.json
+|-- host.json
+```
+
+- The `api` folder contains all the internal and external OpenAPI specification.
+- The `src` folder contain all the implementation of the functions
+
+### src Folder
+
+The `src` folder contains the `main.ts` file that exports the implementation for all the functions defined in this workspace. The Azure Core Function V4 runtime is required to execute it.
+
+#### functions Folder
+
+Contains the implementation of the functions handlers
+
+#### generated Folder
+
+This folder contains the autogenerated code from type definitions in OpenApi specs located into the `api` folder. The command `yarn generate` start the generation process.
+
+#### models Folder
+
+Containst the file for the model layer of the function, that allows to access to databases (Redis and Cosmos for example).
+
+#### middlewares Folder
+
+Contains middlewares for local business logic, that in not usefull to share with other workspaces in the same monorepo but can be shared across multiple functions in the same applicative.
+
+## ENV variables
+
+The following table contains the required ENV variables that the applicative require
+
+| Variable name                      | Description                                  | type    |
+|------------------------------------|----------------------------------------------|---------|
+| APPINSIGHTS_INSTRUMENTATIONKEY     | The Application Insights instrumentation key | string  |
+| FETCH_TIMEOUT_MS                   | (optional) Fetch Timeout for AbortableFetch  | number  |
+
+## Local Execution
```

Comment:
Instead of loading an .env file using dotenv have you considered the built-in support for local envs described here: https://learn.microsoft.com/en-us/azure/azure-functions/functions-develop-local